compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -518,6 +518,7 @@ template <Type indexT> class ArgMaxTestGraph final : public SimpleTestGraph
 public:
   // NOTE: Do not override `luci::CircleNode* input(void)` incidentally
   loco::Node *input_argmax(void) { return _argmax->input(); }
+  loco::Node *dimension(void) { return _argmax->dimension(); }
```

Comment:
Title/desctiption has no relation to this change. Either change the title/description or separate this change.

Reply:
The change has been generated as the pr #6664

Reply:
@seanshpark Sorry for the unnatural English. From my view, I have been focusing on avoiding Korean-English when writing English, as my experience is based on communicating with non-Korean English users, rather than native English users. I will more care about using nature expressions.